Akkadu/rsi-widgets
# Add styling Prop + Doc + to refractor current interpretation player CSS.

---
The original purpose of this task is defined [here](https://www.notion.so/akkadu/RSI-SDK-Add-styling-Prop-Doc-to-refractor-current-interpretation-player-CSS-15ccbf0250454c5190b730192cde4f70) but later we changed its scope in order to make our widget more flexible. So the current purpose of this task is as following.

1. To allow widget users to add custom classes to different parts of the widget so that they can customize it as they want.
2. Remove the css from string and convert it to scss and compile it to a minified css file.
3. Add docs for the new props to add classes and add links for css to the vanilla js examples.

## How To Test
1. From the deploy preview try to add [these classed](https://deploy-preview-17--rsi-akkadu-documentation.netlify.app/interpretation-player/props.html#additional-props) and you should see the new classes added in the browser inspector.
2. pull the branch locally and after running `yarn build` in the root dir you should see a .css file in the dist file of the `rsi-interpretation-player` package.
3. from the `rsi-interpretation-player` package when you run `yarn start:css` it should regenerate the css file in the dist dir and also watch the .scss file for changes.




## Checklist



加油！
